Enterprise-CMCS/macpro-mdct-mfp
# Create commonTests file and testA11y methods

### Description

- Refactor similar accessibility tests to a shared file
- Reorganize tests under a single `describe` in each file with clearer descriptions
  - For component tests, use `<ComponentName />` as the top level description
  - For methods, use `methodName()` as the description
- Use `test` instead of `it` for test declarations

### Related ticket(s)


CMDCT-3777

---
### How to test

1. cd to `macpro-mdct-mfp/services/ui-src`
2. Run `yarn coverage`
3. Verify tests pass and coverage is unaffected

![Screenshot 2024-06-27 at 11 43 37 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/cd996ea3-63f7-4bca-ad2b-cf32b5be29cf)

### Notes


Example Test Output:
  - Current test descriptions:
![Screenshot 2024-06-27 at 11 50 14 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/5846738f-e12f-43f1-88f2-a456ec9e894e)
  - After organizing: 
![Screenshot 2024-06-27 at 11 56 50 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/4f189cc5-6fbd-4a67-8c95-553da9f6a9e1)
---
### Pre-review checklist
